docs/program-setup/bare_metal/baremetal_usage.md

```diff
@@ -0,0 +1,75 @@
+# Using bare metal profile
+
+## Creating a bare metal application
+
+First, clone Mbed OS into the directory where we want to create an application
+```
+mkdir my_mbed_app && cd my_mbed_app
+git clone https://github.com/ARMmbed/mbed-os.git
+```
+
+To enable Mbed OS bare metal, create an `mbed_app.json` with the following contents:
+```
+{
+    "requires": ["bare-metal"]
+}
+```
+
+On bare metal, features are enabled with a bottom-up approach - those listed in [Bare metal APIs](bare_metal_api.md#bare-metal-apis) are now ready for use (but not other features) in our application. See [Bare metal example](baremetal_example.md).
+
+To pull in additional features (take filesystem for example)
+
+1. Locate the libraries we want to use in `mbed-os/` and find their names in their respective `mbed_lib.json`, e.g. `features/storage/filesystem/mbed_lib.json`:
```

Comment:
```suggestion
1. Locate the library you want to use in `mbed-os/`.
1. In the library folder, find the library's name in `mbed_lib.json`. You will need it for the next step.

     For example: `features/storage/filesystem/mbed_lib.json`:
```

Reply:
I have to say I'm not sure about using "library" here, because in some contexts it has a more specific meaning. And what we're really looking for here is either .cpp or .h. 

I think this needs clarifying.